apps/andi/lib/andi/input_schemas/dataset_input.ex

```diff
@@ -47,7 +48,9 @@ defmodule Andi.InputSchemas.DatasetInput do
 
   @key_value_type_keys [:sourceQueryParams, :sourceHeaders]
 
-  @non_embedded_types Map.drop(@types, @key_value_type_keys)
+  @embedded_types @key_value_type_keys
```

Comment:
Assuming that the idea is that this will eventually include the data dictionary type as well?